### ISSUE TITLE
Issue with the latest release (download)

### ISSUE THREAD

**Issue by T3M1N4L** [2775Josh/JAR-Template#13]
When importing into VSCode with the Vex extension there is an error-

[11/25/2024 16:58:40:031] [Command] [error] Import Project Failed: Error: ENOENT: no such file or directory, open 

This is probably because some of the files are not included in the zipped download.

**Comment by ghost** [comment]
I was able to get around the issue by backlogging the two VEX extensions to versions 0.6.0 and 0.2.2 respectively. Once you import and open the project, you can save it as a workspace and update the extensions to the latest versions. Once done, restart the extensions and there shouldn't be any issues moving forward with opening the project or downloading the code to your brain.

I just started tinkering with this so I have no clue if this affects anything down the road but I will reply/update if I do find something.

**Comment by Aaqil-25** [comment]
Yeah, it worked for me too  after changing the VEX Robotics to 0.6.0 and VEX Robotics Feedback to 0.2.2.
